facia/public/facebook_ia_remote.html

```diff
@@ -50,8 +50,8 @@
             }
 
             function exposeMetaDataForKrux() {
-                guardian = { 'config' : {
-                    'page' : {
+                guardian = {
```

Comment:
Try just `config =`, and remove guardian all together. It serves no purpose to have an object within an object. Or you could call it `guardianConfig` or something else if you prefer :smile: